auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -123,7 +124,11 @@ public void onNewUser(User user) {
             FragmentTransaction ft = getSupportFragmentManager().beginTransaction()
                     .replace(R.id.fragment_register_email, fragment, RegisterEmailFragment.TAG);
 
-            if (emailLayout != null) ft.addSharedElement(emailLayout, "email_field");
+
+            if (emailLayout != null) {
+			    ViewCompat.setTransitionName(emailLayout, "email_field");
+			    ft.addSharedElement(emailLayout, "email_field");
+			}
```

Comment:
@clydebarrow Checkstyle is correct: please remove the new line above the if statement and pull back the indentation so that the end bracket is aligned with the `i` in `if` and the code is indented with 4 spaces.